HPXMLtoOpenStudio/resources/EPvalidator.rb

```diff
@@ -602,15 +607,19 @@ def self.run_validator(hpxml_doc)
         'ArrayAzimuth' => one,
         'ArrayTilt' => one,
         'MaxPowerOutput' => one,
-        'InverterEfficiency' => one, # PVWatts default is 0.96
-        'SystemLossesFraction' => one, # PVWatts default is 0.14
+        'InverterEfficiency' => zero_or_one,
+        'SystemLossesFraction | YearModulesManufactured' => zero_or_more,
       },
 
       # [ClothesWasher]
       '/HPXML/Building/BuildingDetails/Appliances/ClothesWasher' => {
         'SystemIdentifier' => one, # Required by HPXML schema
         '[Location="living space" or Location="basement - conditioned" or Location="basement - unconditioned" or Location="garage"]' => one,
-        'ModifiedEnergyFactor | IntegratedModifiedEnergyFactor' => one,
+        'ModifiedEnergyFactor | IntegratedModifiedEnergyFactor' => zero_or_one, # See [ClothesWasher=Detailed]
```

Comment:
For linked properties, try something like this:
`'[ModifiedEnergyFactor | IntegratedModifiedEnergyFactor] | RatedAnnualkWh | LabelElectricRate | LabelGasRate | LabelAnnualGasCost | Capacity' => zero_or_six`